contrib/python-bindings/source/triangulation_wrapper.cc

```diff
@@ -1266,6 +1351,64 @@ namespace python
   }
 
 
+
+  void
+  TriangulationWrapper::set_manifold(const int        number,
+                                     ManifoldWrapper &manifold)
+  {
+    if ((dim == 2) && (spacedim == 2))
```

Comment:
Can you add a check that `manifold` and `triangulation` have the `dim` and `spacedim`